Add breakpad to Docker image

driazati opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #55647 Add minidump collection via breakpad
* **#55439 Add breakpad to Docker image**

This adds [breakpad](https://github.com/google/breakpad) to the build in CI (just on one image for now). I attempted in #54739 to build it from source as a normal third_party submodule but it uses autotools and has some weird build steps that made it hacky to integrate. We really only need it for release builds anyways since its use is moot if built with anything but `RELEASE=1`.

Differential Revision: [D27679766](https://our.internmc.facebook.com/intern/diff/D27679766)

Commit:
Add breakpad to Docker image

[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"




[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"




[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"




[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"




[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"


This adds [breakpad](https://github.com/google/breakpad) to the build in CI (just on one image for now). I attempted in #54739 to build it from source as a normal third_party submodule but it uses autotools and has some weird build steps that made it hacky to integrate. We really only need it for release builds anyways since its use is moot if built with anything but `RELEASE=1`.

[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"


This adds [breakpad](https://github.com/google/breakpad) to the build in CI (just on one image for now). I attempted in #54739 to build it from source as a normal third_party submodule but it uses autotools and has some weird build steps that made it hacky to integrate. We really only need it for release builds anyways since its use is moot if built with anything but `RELEASE=1`.

[ghstack-poisoned]

malfet commented on .circleci/docker/common/install_breakpad.sh:
```diff
@@ -0,0 +1,13 @@
+
+set -ex
+
+git clone https://github.com/google/breakpad.git
```
It would be good to pin to a specific version  here, otherwise we are bound to run into a regressions

Commit:
Update on "Add breakpad to Docker image"


This adds [breakpad](https://github.com/google/breakpad) to the build in CI (just on one image for now). I attempted in #54739 to build it from source as a normal third_party submodule but it uses autotools and has some weird build steps that made it hacky to integrate. We really only need it for release builds anyways since its use is moot if built with anything but `RELEASE=1`.

[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"


This adds [breakpad](https://github.com/google/breakpad) to the build in CI (just on one image for now). I attempted in #54739 to build it from source as a normal third_party submodule but it uses autotools and has some weird build steps that made it hacky to integrate. We really only need it for release builds anyways since its use is moot if built with anything but `RELEASE=1`.

Differential Revision: [D27679766](https://our.internmc.facebook.com/intern/diff/D27679766)

[ghstack-poisoned]

Commit:
Update on "Add breakpad to Docker image"


This adds [breakpad](https://github.com/google/breakpad) to the build in CI (just on one image for now). I attempted in #54739 to build it from source as a normal third_party submodule but it uses autotools and has some weird build steps that made it hacky to integrate. We really only need it for release builds anyways since its use is moot if built with anything but `RELEASE=1`.

Differential Revision: [D27679766](https://our.internmc.facebook.com/intern/diff/D27679766)

[ghstack-poisoned]
